doc/conf.py

```diff
@@ -110,10 +110,11 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+# Don't show version in title
+html_title = "FOSHttpCache Documentation"
```

Comment:
agreed. we could make it "FOSHttpCache Documentation (current)" though,
to make it more obvious, if you want.